www/src/pages/pl/faq.md

```diff
@@ -18,6 +18,7 @@ Jeżeli nie znasz poszczególnych technologi użytych w projekcie, skorzystaj z
 - [Prisma](https://prisma.io)
 - [Tailwind CSS](https://tailwindcss.com)
 - [tRPC](https://trpc.io)
+- [Drizzle](https://drizzle.team/)
```

Comment:
If create-t3-app is using drizzle-orm itself, I think it would be better to have a link to drizzle-orm docs rather than the drizzle team. I guess this one should be much better: https://orm.drizzle.team/docs/overview. We have also updated an overview page and the documentation itself.

Reply:
Awesome, we'll add proper docs later so we can adress that then I suppose